tools/repo/notify.py

```diff
@@ -151,6 +170,17 @@ async def oncall_string(self):
                     return component.get("summary")
         return "unable to find this week's oncall"
 
+    @async_property(cache=True)
```

Comment:
```suggestion
    @async_property
```

i think this is only called once - so no need to cache result